emcc.py

```diff
@@ -650,6 +650,9 @@ def check_human_readable_list(items):
     extras = shared.Settings.BINARYEN_EXTRA_PASSES.split(',')
     passes += [('--' + p) if p[0] != '-' else p for p in extras if p]
 
+  if shared.Settings.INSTRUMENT:
```

Comment:
I think this BINARYEN_EXTRA_PASSES should comes last go maybe move this up a few lines

Reply:
From our discussion last week, I believe the logging function will be hard-coded in Wasm so this will no longer be necessary. The instrumentation phase will also be executed from wasm-split so I think the code will be something like:
if shared.Settings.INSTRUMENT:
  wasm-split instrument a.wasm ...

And the wasm-split tool should preferably be run at the end so that both instrumentation and splitting will run on the same binary?